core/includes/theme.inc

```diff
@@ -2265,7 +2290,8 @@ function theme_table($variables) {
         $output .= " </tr>\n";
       }
     }
-    $output .= "</tbody>\n";
+
+    $output .= "</" . $tag . ">\n";
```

Comment:
Same as previous comment, this could be:

```suggestion
    $output .= "</$tag>\n";
```

Unless we wanna keep as much code parity as D7.